backend/tests/test_routes/test_orders.py

```diff
@@ -88,3 +88,28 @@ def test_read_order_detail_one(session, client):
 def test_read_order_detail_none(client):
     response = client.get(f"{ROOT_PATH}/orders/-123")
     assert response.status_code == 404
+
+
+def test_create_order_no_auth(client):
+    response = client.post(
+        f"{ROOT_PATH}/orders/", json={"order_constraints": {"gmpe": ["Abrahamson"]}}
+    )
+    assert response.status_code == 401
```

Comment:
So, just for completeness sake: this means not sending along X-APIKEY will cause a 401 now, right?
Fine by me!

Reply:
Yes.

Reply:
Souds alright - Merging!